src/appchooserdialog.c

```diff
@@ -386,11 +390,11 @@ app_chooser_dialog_new (const char **choices,
       gtk_label_set_label (GTK_LABEL (dialog->heading), _("Choose an application."));
     }
 
-  dialog->choices = g_strdupv ((char **)choices);
-  n_choices = g_strv_length ((char **)choices);
+  ensure_default_in_initial_list (choices, default_id);
 
-  ensure_default_is_below (dialog->choices, default_id, MIN (n_choices, INITIAL_LIST_SIZE));
+  dialog->choices = g_strdupv ((char **)choices);
 
+  n_choices = g_strv_length ((char **)choices);
```

Comment:
Space after the cast.

Reply:
OK.

Reply:
Sorry, let's change this in a follow-up because the surrounding code does not use casts here. It feels weird to contradict that.